yt_dlp/extractor/bfi.py

```diff
@@ -29,6 +30,6 @@ def _real_extract(self, url):
             if not ooyala_id:
                 continue
             entries.append(self.url_result(
-                'ooyala:' + ooyala_id, 'Ooyala',
+                'ooyala:' + ooyala_id,
                 ooyala_id, player_attr.get('data-label')))
```

Comment:
```suggestion
                'ooyala:' + ooyala_id, video_id=ooyala_id, video_title=player_attr.get('data-label')))
```

Reply:
imo, we dont need to change this since the site is _WORKING=False anyways. All it changes is the error message, but the "not working" message is issued anyway so error dont matter